src/core/store/middleware.ts

```diff
@@ -5,7 +5,11 @@ import { ActionCreators } from 'redux-undo';
 import * as validatorMiddleware from 'redux-validator';
 import FluxCapacitor from '../../flux-capacitor';
 import Actions from '../actions';
+import Creators from '../actions/creators';
```

Comment:
can we import this as `ActionCreators` just for clarity

Reply:
Conflicts with ActionCreators from redux-undo

Reply:
then we can rename the `redux-undo` one